ext/libjpeg.cmd

```diff
@@ -1,7 +1,10 @@
 : # If you want to use a local build of jpeg, you must clone the repos in this directory first,
-: # then enable CMake's AVIF_LOCAL_JPEG.
-: # This git tag isn't likely to move much, as libjpeg isn't actively developed anymore.
+: # then set CMake's AVIF_JPEG=LOCAL.
+: # The git tags below are known to work, and will occasionally be updated. Feel free to use a more recent commit.
```

Comment:
Nit: There is only one git tag in this file, so "The git tag below is known to work" sounds better.